administration/src/util/translations/de.json

```diff
@@ -56,7 +56,7 @@
     "amountOfWork": "Arbeitsstunden pro Woche (Durchschnitt)",
     "workSinceDate": "Tätig seit",
     "isAlreadyVerified": "Die Angaben wurden bereits durch den zuständigen Verein bei Beantragung verifiziert",
-    "payment": "Für diese ehrenamtliche Tätigkeit wurde eine Aufwandsentschädigung gewährt, die über den jährlichen Freibetrag hinaus geht (840 Euro Ehrenamtspauschale bzw. 3000 Euro Übungsleiterpauschale):",
+    "payment": "Für diese ehrenamtliche Tätigkeit wurde eine Aufwandsentschädigung gewährt, die über den jährlichen Freibetrag hinaus geht (840 Euro Ehrenamtspauschale bzw. 3000 Euro Übungsleiterpauschale)",
```

Comment:
Otherwise the ":" symbol is duplicated
![image](https://github.com/user-attachments/assets/a595bfc9-3a43-4df2-b9d8-941922001906)

Reply:
I think it might be good to do this the other way round though, keep the symbol here and remove it otherwise. The reason for this is that some languages handle punctuation marks differently. For example, in French there is a space between the sentence and the colon, e.g. `C'est vrai : j'ai ...`.
Even though we don't have any languages atm that do this, I think it would not hurt.

Reply:
Would make sense, but this translation is also used on the application form, where ":" is not needed at all:
![image](https://github.com/user-attachments/assets/bd176ba6-c1b2-4214-aa29-a8628cadd766)

Reply:
okay, then lets just keep it like this :)